.travis.yml

```diff
@@ -11,35 +11,23 @@ python:
 addons:
   postgresql: "9.6"
   apt:
-    sources:
-      - pov-wkhtmltopdf
```

Comment:
This changes were already introduced in https://github.com/OCA/pos/commit/3386c81e34d92455d04b697c54e6579f6c9828fe

Just rebase your branch to get the latest commits

Reply:
I just rebased.